services/web3id-issuer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
+
+- Added graceful shutdown
```

Comment:
We would typically have "Unreleased changes" section, and then only add the version once we decide to release.

But it's OK for this now.